### DISCUSSION TITLE
Production deployment issue by using docker

### DISCUSSION THREAD

**Discussion by andriiradkevych** [freshframework/fresh#2080] [General]
https://fresh.deno.dev/docs/concepts/deployment - used this article for trying to deploy my deno prod build on docker

**Dockerfile**
```
FROM denoland/deno:1.35.0

ARG GIT_REVISION
ENV DENO_DEPLOYMENT_ID=${GIT_REVISION}

WORKDIR /app

COPY . .
RUN deno cache main.ts

EXPOSE 8000

CMD ["run", "-A", "main.ts"]
```

1. I've done build by command `deno run -A dev.ts build` 
2.  Started my docker build by command `docker build --build-arg GIT_REVISION=$(git rev-parse HEAD) -t fresh-production .`

Get next issue:

<img width="563" alt="Screenshot 2023-11-28 at 15 29 42" src="https://github.com/denoland/fresh/assets/26725494/854bea7c-628c-46fb-8f76-86be551a50b5">

I understand that I get it because of this line ` import data from 'data/pages/home.json' with { type: "json" };` , but it is eligible code for `Deno` , what can I do to fix it ?

**Comment by marvinhagemeister** [comment]
The deno version `1.35` is too old and doesn't support the newer import attribute spec. Update the version number in the docker file to `1.38.3` to resolve this issue.

EDIT: Updated the docs.
